app/components/PhysicsPlane.js

```diff
@@ -0,0 +1,42 @@
+import {
+  MeshPhongMaterial
+} from 'three';
+
+import {
+  PlaneModule
+} from 'physics-module-ammonext';
+
+// Meshes
+import {
+  Plane
+} from '@whs+meshes';
+
+import {MeshComponent} from '@whs/core/MeshComponent';
+
+export class PhysicsPlane extends Plane {
+  /**
+   * @name PhysicsPlane
+   * @param params
+   */
+  constructor(params = {}) {
+
+    params.modules = [
+      new PlaneModule({
```

Comment:
But, anyway, that's just my suggestion of how to do it properly. For very specific component you can ignore that. But if you want to make your components universal - use the code above.